detection_rules/rule.py

```diff
@@ -698,6 +698,9 @@ class EQLRuleData(QueryRuleData):
     """EQL rules are a special case of query rules."""
     type: Literal["eql"]
     language: Literal["eql"]
+    timestamp_field: Optional[str]
+    event_category_override: Optional[str]
+    tiebreaker_field: Optional[str]
```

Comment:
Do we want to add a `min_compat` to these?

Reply:
I checked back to `8.0` and it appears these are all still valid fields for EQL rules.